android/src/main/java/io/ltbl/bdkrn/Utils.kt

```diff
@@ -0,0 +1,53 @@
+package io.ltbl.bdkrn
+import android.util.Log
+import org.bitcoindevkit.*
+
+object ProgressLog : Progress {
+    override fun update(progress: Float, message: String?) {
+        Log.i(progress.toString(), "Progress Log")
+    }
+}
+
+fun createChangeDescriptor(descriptor: String): String {
+    return descriptor.replace("/84'/1'/0'/0/*", "/84'/1'/0'/1/*")
```

Comment:
It's possible that this will change in a coming commit but I would like to bring this to your attention.

I was just testing the current version of bdk-rn in our production app and I noticed that the derivation path here is always for bitcoin testnet

As per [BIP44](https://en.bitcoin.it/wiki/BIP_0044#Registered_coin_types), coin type 0 should be used for bitcoin network and type 1 for testnet

Reply:
Thanks, @Czino Yes this will change. Once updated it will use a network argument to create a testnet/mainnet descriptor.

It will work like as described here: https://pub.dev/documentation/bdk_flutter/latest/bdk_flutter/Descriptor-class.html